OrbitGl/CaptureSerializerLoadFuzzer.cpp

```diff
@@ -42,7 +42,7 @@ int LLVMFuzzerTestOneInput(uint8_t* buf, size_t len) {
 }
 
 int LLVMFuzzerInitialize(int*, char***) {
-  OrbitApp::Init({});
+  OrbitApp::Init({}, nullptr);
```

Comment:
not sure if this is right way - seems like main_thread_executor is not used in this case but I might be wrong.